### PR TITLE
SAK-33996 display the overridden letter course grade with the student's calculated percent next to it. 

### PR DESCRIPTION
**Pull request by ottenhoff**
Old behavior displayed a percent correlated with the new overridden letter grade.

### CODE REVIEW

File: gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/util/CourseGradeFormatter.java [hunk 1]
```diff
@@ -132,20 +132,7 @@ private String build(final CourseGrade courseGrade) {
...
-			Double mappedGrade = this.gradebook.getSelectedGradeMapping().getGradeMap().get(courseGrade.getEnteredGrade());
-			if (mappedGrade == null) {
-				mappedGrade = new Double(0);
-			}
-			calculatedGrade = FormatHelper.formatDoubleAsPercentage(mappedGrade);
-
-		} else {
-			calculatedGrade = FormatHelper.formatStringAsPercentage(courseGrade.getCalculatedGrade());
-		}
+		final String calculatedGrade = FormatHelper.formatStringAsPercentage(courseGrade.getCalculatedGrade());
```
Comment (adrianfish) [inline gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/util/CourseGradeFormatter.java]: So, does mappedGrade end up being set at the bottom of the grade boundary, and that was the problem? Does formatStringAsPercentage pick the middle of the grade range?

Comment (ottenhoff) [reply gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/util/CourseGradeFormatter.java]: Right, old functionality would find the percentage corresponding to the overridden grade which would confuse instructors.

New behavior just uses the student's calculated grade. So you would see C (59.01%) bolded indicating the instructor was nice and gave you a C but your calculated grade was only 59.01%

Comment (adrianfish) [reply gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/util/CourseGradeFormatter.java]: So, enteredGrade remains the override that the instructor has applied and calculatedGrade is just as it should be, the calculated grade. Sounds good to me.

Comment (ottenhoff) [reply gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/util/CourseGradeFormatter.java]: Yes, this is what was decided by T&L as desired behavior. Seems reasonable

**Pull request merged**